diff -urN transmission-2.82.orig/libtransmission/fdlimit.c transmission-2.82/libtransmission/fdlimit.c
--- transmission-2.82.orig/libtransmission/fdlimit.c	2013-08-09 11:45:40.882416172 +0900
+++ transmission-2.82/libtransmission/fdlimit.c	2013-10-15 10:55:05.699781757 +0900
@@ -520,15 +520,15 @@
       /* set the open-file limit to the largest safe size wrt FD_SETSIZE */
       if (!getrlimit (RLIMIT_NOFILE, &limit))
         {
-          const int old_limit = (int) limit.rlim_cur;
-          const int new_limit = MIN (limit.rlim_max, FD_SETSIZE);
-          if (new_limit != old_limit)
-            {
-              limit.rlim_cur = new_limit;
-              setrlimit (RLIMIT_NOFILE, &limit);
-              getrlimit (RLIMIT_NOFILE, &limit);
-              tr_logAddInfo ("Changed open file limit from %d to %d", old_limit, (int)limit.rlim_cur);
-            }
+//XXX curl7.12 //          const int old_limit = (int) limit.rlim_cur;
+//XXX curl7.12 //          const int new_limit = MIN (limit.rlim_max, FD_SETSIZE);
+//XXX curl7.12 //          if (new_limit != old_limit)
+//XXX curl7.12 //            {
+//XXX curl7.12 //              limit.rlim_cur = new_limit;
+//XXX curl7.12 //              setrlimit (RLIMIT_NOFILE, &limit);
+//XXX curl7.12 //              getrlimit (RLIMIT_NOFILE, &limit);
+//XXX curl7.12 //              tr_logAddInfo ("Changed open file limit from %d to %d", old_limit, (int)limit.rlim_cur);
+//XXX curl7.12 //            }
         }
     }
 }
